holidays/locale/en_IN/LC_MESSAGES/XNSE.po

```diff
@@ -0,0 +1,97 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+# SOME DESCRIPTIVE TITLE
+# This file is distributed under the same license as the Holidays package.
+# FIRST AUTHOR saheelsapovadia23@gmail.com, 2025.
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.79\n"
```

Comment:
Once you've update the main file, just run `make l10n` again to automatically populate the code comments
```suggestion
"Project-Id-Version: Holidays 0.80\n"
```